docs/4.0/components/breadcrumb.md

```diff
@@ -32,3 +32,22 @@ Similar to our navigation components, breadcrumbs work fine with or without the
   <span class="breadcrumb-item active">Bootstrap</span>
 </nav>
 {% endexample %}
+
+
+## Regarding accessibility
+
+If you’re using breadcrumbs to provide a navigation, be sure to add a `aria-label="breadcrumb"` to describe the type of navigation provided in the `<nav>` element.
```

Comment:
Should we be doing this in our examples?

Reply:
I think that if we can deliver accessible snippets from bootstrap we will be doing a huge dent to help people with disabilities.

So even though the other examples will render correctly, I think we should deliver just one example that follows an accessible pattern.

I'll change this PR and send a new option